config/option.go

```diff
@@ -14,8 +14,8 @@ type OptionDefinition struct {
 	Validate    func(value interface{}) error
 }
 
-// RegisterConfigOption takes in the name of the command and an OptionDefinition object
-func RegisterConfigOption(command string, options map[string]OptionDefinition) {
```

Comment:
Make sure that you've renamed it throughout the application, seems like travis found a place were it wasn't :)